tests/test_requirements/aws_test_requirements.txt

```diff
@@ -0,0 +1,4 @@
+runhouse[aws]
```

Comment:
similarly, can take out runhouse[aws] and explicitly call pip install runhouse[aws] in the workflow.